### PR TITLE
Fix README cargo.toml dependency github URL

### PR DESCRIPTION
**Pull request by ttrssreal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ For now the crate can be included by branch or by tag via git.
 
 ```toml
 #[dependencies]
-unicode-categories = { git = "https://github.com/ncatelli/unicode_categories", branch = "main" }
+unicode-categories = { git = "https://github.com/ncatelli/unicode-categories.git", branch = "main" }
```
Comment (ncatelli) [inline README.md:11]: This should work essentially equivalently to how it was before. Was this failing for you?

Comment (ttrssreal) [reply README.md:11]: Yep

Comment (ncatelli) [reply README.md:11]: Oh i see `_` vs `-`. Good catch. Thank you.

Comment (ttrssreal) [reply README.md:11]: Oh I didn't see the `.git`, sorry for the confusion 😅

**Pull request merged**